src/views/virtualmachines/details/tabs/details/components/grid/rightGrid/VirtualMachineDetailsRightGridLayout.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
```

Comment:
Should the directory name be `right-grid` instead of `rightGrid`?

Reply:
I think it would be best once we all finish our work for this version, we fix all directories structure and naming to follow our coding conventions, and to start our consistency from there